apps/server/src/shared/testing/map-user-to-current-user.ts

```diff
@@ -1,14 +1,15 @@
 import { ObjectId } from '@mikro-orm/mongodb';
 
-import { Account, User } from '@shared/domain';
+import { Account, EntityId, User } from '@shared/domain';
 import { ICurrentUser } from '@src/modules/authentication';
 
-export const mapUserToCurrentUser = (user: User, account?: Account): ICurrentUser => {
+export const mapUserToCurrentUser = (user: User, account?: Account, systemId?: EntityId): ICurrentUser => {
 	const currentUser: Partial<ICurrentUser> = {
 		userId: user.id,
 		roles: user.roles.getItems().map((r) => r.id),
 		schoolId: user.school.id,
 		accountId: account ? account.id : new ObjectId().toHexString(),
+		systemId,
```

Comment:
Note that one PR is also touched this area, i see that we have differents between test data and production for now. Please check that they are in sync after the PR 4021 is merged.

https://github.com/hpi-schul-cloud/schulcloud-server/pull/4021/files

That we run in differents is a good hint, that we should use the same mapping for test and prod. 

I also note that it exist some  "as ICurrentUser" in test. They should be possible to avoid over this mapper.